pilot/pkg/model/context.go

```diff
@@ -491,6 +491,9 @@ type NodeMetadata struct {
 	// Generator indicates the client wants to use a custom Generator plugin.
 	Generator string `json:"GENERATOR,omitempty"`
 
+	// DNSCapture indicates whether the workload has enabled dns capture
+	DNSCapture string `json:"DNS_CAPTURE,omitempty"`
+
```

Comment:
is this a feature flag or long term configuration mechanism?

Reply:
For now, this will be a per sidecar flag (which like SDS can be enabled globally via the injector or on VMs). Once the entire thing is stable (i.e. no DNS issues on pods) we can get rid of this flag and make this the defacto one.